### PR TITLE
Document custom Jinja template filters

### PR DESCRIPTION
**Pull request by happz**
Pull Request Checklist

* [x] implement the feature
* [x] write the documentation

### CODE REVIEW

**Comment by martinhoyer** [comment]
Is Overview the right place for it? Looking at dictionary, overview = "a general explanation or description of something".

fwiw, I still think the man page should be dropped :) 
On the other hand, how about adding [tldr](https://github.com/tldr-pages/tldr) entry?

**Comment by happz** [comment]
> Is Overview the right place for it? Looking at dictionary, overview = "a general explanation or description of something".

It's not the right place, and trying to sneak it there was a stupid idea from the beginning. Now it's under `Code`, https://tmt--2769.org.readthedocs.build/en/2769/code/template-filters.html

**Review by lukaszachy — APPROVED** [review]

File: docs/code/index.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ the documentation generated from sources linked below.
 .. toctree::
     :maxdepth: 2
 
+    Template Filters <template-filters>
```
Comment (psss) [inline docs/code/index.rst:16]: Would you mind moving it down one line? I'd say that `Class Overview` is something very basic / essential and could be listed as the very first section.

File: pyproject.toml [hunk 2]
```diff
@@ -182,7 +182,7 @@ man = [
     "sed '/versionadded::/d' -i {root}/man.rst",
     "rst2man.py {root}/man.rst > {root}/tmt.1",
     "rm -f {root}/man.rst",
-    ]
+]
```
Comment (psss) [inline pyproject.toml:185]: Let's keep the hanging indent consistent across the file?

**Review by psss — APPROVED** [review]
Nice improvement, thanks! Just two nitpicks.

File: docs/code/index.rst (hunk 1 above)
Comment (happz) [reply docs/code/index.rst]: Moved in 31f63b84

File: pyproject.toml (hunk 2 above)
Comment (happz) [reply pyproject.toml]: Fixed in 31f63b84

**Review by therazix — APPROVED** [review]

**Pull request merged**